### ISSUE TITLE
Including raw data in repo

### ISSUE THREAD

**Issue by BramVanroy** [UniversalDependencies/UD_English-EWT#190]
Hello

I am wondering why the raw data (TXT) is not included in the UD repositories (not just in this one). To get the raw data it seems required to download the whole release [here](https://lindat.mff.cuni.cz/repository/xmlui/handle/11234/1-3424#show-files). This has the disadvantage that you need to download the whole UD treebank and can't pick a specific language let alone a specific treebank. Furthermore you can only dynamically get that dataset with curl which may not be easy to maintain compared to git.

I would like to have access to the raw files through git for development and evaluation reasons. During development of new UD parsers, it is useful to benchmark the parser's performance against the gold .conllu test set with an [evaluation script](https://universaldependencies.org/conll18/evaluation.html). To generate output with the parser, however, you need access to the raw data. As said, the power of git exceeds that of curl and, especially, it is cumbersome to have to download the whole treebank.

Is it possible to add the raw data to the repositories as well? Or, if not, is their a particular reason that they are not added?

Thank you

**Comment by nschneid** [comment]
@dan-zeman is the infrastructure expert, but raw sentences are available in the `# text = ` lines of the .conllu files. Is there something more that is needed for the .txt format?

**Comment by BramVanroy** [comment]
@nschneid That is true, but the conllu files are already sentence segmented whereas the raw data contains paragraphs. So to evaluate the sentence segmentation capabilities of a model, it is useful to have the raw paragraphs to see how well a model can segment the text. Perhaps one could try to glue together the relevant sentences into a paragraph by using the identifiers but that seems cumbersome and error prone compared to having access to the raw data directly.

I don't think it is a matter of size restrictions since the raw files are 10x smaller than the conllu files.

**Comment by dan-zeman** [comment]
Moving this issue to the docs repository, as it pertains to all UD treebanks.
